frog/package.json

```diff
@@ -53,8 +55,9 @@
     "sharedb-mongo": "^1.0.0-beta.4",
     "styled-components": "^2.0.0",
     "websocket-json-stream": "0.0.3",
-    "ws": "^3.0.0",
-    "ac-induction": "*"
+    "ws": "^3.0.0"
   },
-  "devDependencies": {"flow-bin": "^0.48.0"}
-}
\ No newline at end of file
+  "devDependencies": {
+    "flow-bin": "^0.48.0"
+  }
+}
```

Comment:
Thank you for sorting `frog/package.json` in alphabetical order. Did it happen when you ran `meteor add ...` ?